pkg/virtual/framework/fixedgvs/register.go

```diff
@@ -36,14 +46,28 @@ func (vw *FixedGroupVersionsVirtualWorkspace) Register(rootAPIServerConfig gener
 			GenericConfig: &genericapiserver.RecommendedConfig{Config: *rootAPIServerConfig.Config, SharedInformerFactory: rootAPIServerConfig.SharedInformerFactory},
 			ExtraConfig: apiserver.ExtraConfig{
 				GroupVersion:    groupVersionAPISet.GroupVersion,
-				AddToScheme:     groupVersionAPISet.AddToScheme,
 				StorageBuilders: make(map[string]func(apiGroupAPIServerConfig genericapiserver.CompletedConfig) (restStorage.Storage, error)),
 			},
 		}
 		for resourceName, builder := range restStorageBuilders {
+			builder := builder
```

Comment:
is this needed? The builder is a func, isn't it?

Reply:
You're right, it doesn't seem to be needed.

Reply:
Fixed in rebased commits